src/routers/documentation.ts

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import swaggerUi from 'swagger-ui-express';
+import documentation from '../openapi.json';
+import type { JsonObject } from 'swagger-ui-express';
+
+const documentationRouter = express.Router();
+
+// Attempting to satisfy this not being any, but this only pushes back the issue.
+const docs: JsonObject = documentation;
```

Comment:
The lint error is because we can't just cast objects in this way since documentation COULD be anything.

Unfortunately we'll need to write a type guard (maybe swagger-ui-express provides one, but if not, it falls on us unfortunately.  I'll take a look.

Reply:
(taking on this fix now since we def want this as part of the demo)